package.json

```diff
@@ -80,15 +80,18 @@
   },
   "scripts": {
     "clean:build": "rimraf build/*",
-    "clean:dist": "rimraf dist/*",
+    "clean:dist": "rimraf dist/* dist-commonjs/*",
     "clean:docs": "rimraf build/docs/*",
     "clean:icons": "rimraf build/icons/* src/components/Icon/icons/*.tsx",
     "clean:illustrations": "rimraf build/illustrations/* src/components/Illustration/illustrations/*.tsx",
     "clean:svg": "run-p clean clean:icons clean:illustrations",
     "clean": "run-p clean:build clean:dist",
-    "build:css:global": "postcss src/css/index.css --output dist/yamui-base.css",
-    "build:css:components": "postcss \"src/components/**/*.css\" --dir \"dist/components\" --base \"src/components\"",
-    "build:ts:components": "tsc --project .",
+    "build:es2015:css:global": "postcss src/css/index.css --output dist/yamui-base.css",
+    "build:es2015:css:components": "postcss \"src/components/**/*.css\" --dir \"dist/components\" --base \"src/components\"",
+    "build:es2015:ts:components": "tsc --project tsconfig.json",
+    "build:commonjs:css:global": "postcss src/css/index.css --output dist-commonjs/yamui-base.css",
+    "build:commonjs:css:components": "postcss \"src/components/**/*.css\" --dir \"dist-commonjs/components\" --base \"src/components\"",
+    "build:commonjs:ts:components": "tsc --project tsconfig.commonjs.json",
     "build": "run-s clean build:**",
```

Comment:
we could save a few seconds by just copying CSS files from dist to dist-commonjs, but not an issue

Reply:
Trrrue, but then they depend on order of execution. I'll wait for @chrismohr to kill all the CSS :trollface: